boards/rapidsilicon/virgo_proto/virgo_proto.dts

```diff
@@ -37,6 +37,24 @@
  };
  
  &cpu0 {
-	 clock-frequency = <26666667>;
+	clock-frequency = <26666667>;
+ };
+
+ &spi0 {
+	status = "okay";
+	clock-frequency = <26666667>;
+	mx25u16: qspi-nor-flash@0 {
```

Comment:
Change this node label to the one corresponding soc prototype flash.

Reply:
Done